packages/strapi-plugin-content-type-builder/admin/src/containers/App/saga.js

```diff
@@ -26,14 +31,36 @@ export function* getData() {
   }
 }
 
-export function* deleteModel({ modelName }) {
+export function* deleteModel({
+  context: { plugins, updatePlugin },
```

Comment:
Well I think it raises a question deeper than what should we send to the saga, I believe from the above comments that we should really start to think about that internal menu api because currently to update the main left menu we are using some properties of the content manager plugin, which is not a good practice.

So, until we implement that API I think that we can still use the code from above.

> And so to send just the needed data in TableListRow/index.js by the onDelete() func and to avoid passing the whole context object by actions.js deleteModel()

The TableListRow (which is the only component I kept from the old plugin) uses the old context API (contextTypes). And it passes in its context only the `updatePlugin` function and the `plugins` object.